lib/msal-browser/test/interaction_client/RedirectClient.spec.ts

```diff
@@ -2553,235 +2553,6 @@ describe("RedirectClient", () => {
                 );
         });
 
-        it("doesnt navigate if onRedirectNavigate returns false", (done) => {
```

Comment:
Just want to double check: do we have these tests somewhere for using the PCA config option?

Reply:
Yes, it's further down in the file

Reply:
You're right, it's for acquireToken. I'll add some logout tests

Reply:
They're added back now